docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_webhook.go

```diff
@@ -161,7 +161,8 @@ func (r *CronJob) validateCronJob() error {
 /*
 Some fields are declaratively validated by OpenAPI schema.
 You can find kubebuilder validation markers (prefixed
-with `// +kubebuilder:validation`) in the [API](api-design.md)
+with `// +kubebuilder:validation`) in the
+[Desiging an API](api-design.md) section.
```

Comment:
nit: misspelling
```suggestion
[Designing an API](api-design.md) section.
```

Reply:
Thanks for the catch! Embarrassing how I missed this given the size of the PR... 🤭